enterprise/backend/src/metabase_enterprise/sso/settings.clj

```diff
@@ -298,3 +298,23 @@ using, this usually looks like `https://your-org-name.example.com` or `https://e
   :visibility :public
   :setter     :none
   :getter     (fn [] (or (saml-enabled) (jwt-enabled))))
+
+(defsetting ldap-sync-user-attributes
+  (deferred-tru "Should we sync user attributes when someone logs in via LDAP?")
+  :type    :boolean
+  :default true
+  :audit   :getter)
+
+;; TODO - maybe we want to add a csv setting type?
```

Comment:
Do you think it's worth adding whoever wrote the comment's name?

Maybe something worth discussing at the guild sync